tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3494,96 +3477,14 @@ public function deleteGroup(string $group):void {
 	 * @throws GuzzleException
 	 */
 	public function groupHasBeenDeleted(string $group):void {
-		if (OcisHelper::isTestingWithGraphApi()) {
-			$this->graphContext->adminHasDeletedGroupUsingTheGraphApi($group);
+		if ($this->groupExists($group) && $this->isTestingWithLdap()) {
```

Comment:
```suggestion
		if ($this->isTestingWithLdap()) {
```

Reply:
@nabim777 Please, use this suggestion. We want to check if testing-with-Ldap and the defaults to graph-api